frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js

```diff
@@ -1,9 +1,79 @@
 import React from "react";
+import ProminentAppBar from "../../header/ProminentAppBar";
+import {makeStyles} from "@material-ui/core/styles";
+import myTheme from "../../../styling/muiTheme";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import Footer from "../../footer/Footer";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+    },
+    main: {
+        display: 'flex',
+        flexDirection: 'column',
+        minHeight: '70vh',
+    },
+    nannywaytypo: {
+        fontFamily: "Finger Paint",
+    },
+    basictypo: {
+        fontFamily: "Open Sans",
+    },
```

Comment:
same for these, I don't see them used anywhere on this page, did you add these for future components?

Reply:
Gave the classes some use ;)